### PR TITLE
Nblar shopping cart easy

### PR DESCRIPTION
**Pull request by nblar**

### CODE REVIEW

**Comment by girlscript-blr** [comment]
Please follow the below step for putting your solution in the right folder.
The codes/files should be inside a single folder following a specific naming convention. The solution folder name should be `<your GitHub Id>_<your name>`

**Comment by girlscript-blr** [comment]
Hello @nblar ,

This is to inform that, you will not receive your badge if your PR is merged after **31st July 2020**.
Please update the requested change requests before 31st July.

Thanks & Regards!

File: Easy/1. Shopping Cart Problem/solutions/nblar/Opensource.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+              
+              System.out.println("-------------- BILL -------------------");
+              System.out.println("Shop name:- GadgetifyWithGSBlr");
+              System.out.println("Shop address: 311/5 Akshay nagar, Bangalore, Karnataka, India");
+              System.out.println("Shop contact no: +91 9988776655");
+              System.out.println("Customer Details are --------------");
+              System.out.println("Name:- "+name+"\nPhone Number:- "+phoneNo);
+              sum=sum+(0.06*sum);
+              System.out.println("Total Amount of purchase is:- "+sum);
+              System.out.println("Payemnt Mode:- "+payment);
```
Comment (girlscript-blr) [inline Easy/1. Shopping Cart Problem/solutions/nblar/Opensource.java]: `Payment` spelling mistake.

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
Please rename the folder name as `<github username>_<your name>`

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
1. The program should not exit after the shopping list is displayed.
image
2. I'm not getting the purchase list & date and time of purchase in the bill.
image

3. The user should be able to buy only one item.

**Pull request closed without merging**